runtime/io.c

```diff
@@ -505,7 +563,8 @@ static struct custom_operations channel_operations = {
 CAMLexport value caml_alloc_channel(struct channel *chan)
 {
   value res;
-  chan->refcount++;
+  /* prevent finalization during next alloc */
```

Comment:
This comment was found to be misleading in previous review of this code on trunk.  Better remove it (again).